.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
```

Comment:
More an alignment topic (unimportant one): we should have a template .gitignore to copy and not got annoyed :).